### ISSUE TITLE
`modifyPath` missing from 7.1.4 npm package

### ISSUE THREAD

**Issue by wbadart** [selfrefactor/rambda#640]
Hey team-

I noticed in the CHANGELOG that 7.1.0 added `modifyPath` (f1c87de64aed1d6e7160d1ab4a06f395dcd71842) (which I was very excited to see), but the function doesn't actually seem to be present, at least not when Rambda is installed from npm:

```shell
$ mkdir foo && cd foo
$ npm install rambda@7.1.4

added 1 package, and audited 2 packages in 1s

found 0 vulnerabilities

$ node
Welcome to Node.js v16.15.0.
Type ".help" for more information.
> const R = require('rambda')
undefined
> R.modifyPath
undefined
> R.assoc  // just to make sure it's actually rambda
[Function (anonymous)]
```

It seems the module hasn't found its way to `src/` yet, so maybe this is a CI/CD bug that's omitting this from the codegen step (or skipping that step altogether)?

**Comment by selfrefactor** [comment]
Thanks for pointing out this mistake. For some reason, I placed this method as `Rambdax` method and that is why it wasn't exported. It will be fixed with `7.2.0`.

**Comment by selfrefactor** [comment]
Closing the issue as fix was released with `7.2.0`

**Comment by wbadart** [comment]
Thanks!

![image](https://user-images.githubusercontent.com/8717474/181936151-37d03ca9-516a-43a0-a726-5836102ead60.png)
